integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/partition/CarbonAlterTableDropHivePartitionCommand.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.execution.command.partition
 
 import java.util
+import java.util.UUID
```

Comment:
Do you allow dropping of partitions directly on aggregate table? I mean not through parent table.

Reply:
Not allowed